ncclient/transport/ssh.py

```diff
@@ -46,7 +48,6 @@
 BUF_SIZE = 4096
```

Comment:
should this be left here?

Reply:
Yes, this is commonplace. I find it good to be here.